### DISCUSSION TITLE
Next Js Project deploy to CPanel

### DISCUSSION THREAD

**Discussion by suaterkilic** [vercel/next.js#22076] [Help]
Hello,

I created Next js project and I want to deploy it to CPanel. I have VDS Server. How can i do this? Node Js installed on my server.
Thank you.
Best regards

**Comment by krystianjj** [comment]
You can do this in several ways. Copy source to server and build, or copy builded page to server and run via node, but I recommend using pm2 process manager.

**Reply by suaterkilic** [reply]
Thank you for your reply. Can you explain a little more? I need road map :(

**Reply by krystianjj** [reply]
You want deploy static site or dynamic site? Because when you want deploy static site just copy `out` directory to `public_html` but if you want deploy dynamic site and you must running by nodejs it's not so easy.

**Reply by suaterkilic** [reply]
I dont need static html. I need dynamic site. I deploy it to vercel as dynamic but i want deploy to my server. I couldnt found source.

**Reply by krystianjj** [reply]
You must configure nginx and run application. This tutorial can help you -> https://micko.dev/post/how-to-start-your-project-using-systemd-service
